### PR TITLE
Moved functiondetails and matched percentage to top bar if inverted

### PR DESCRIPTION
**Pull request by rgruner**
Change inverted mode to look like the following:
image

### CODE REVIEW

File: flamegraph.pl [hunk 1]
```diff
@@ -1256,12 +1256,18 @@ sub flow {
 $im->filledRectangle(0, 0, $imagewidth, $imageheight, 'url(#background)');
 $im->stringTTF("title", int($imagewidth / 2), $fontsize * 2, $titletext);
 $im->stringTTF("subtitle", int($imagewidth / 2), $fontsize * 4, $subtitletext) if $subtitletext ne "";
-$im->stringTTF("details", $xpad, $imageheight - ($ypad2 / 2), " ");
 $im->stringTTF("dependencies", $xpad, $imageheight - ($ypad2 / 2) + 20, " ");
-$im->stringTTF("unzoom", $xpad, $fontsize * 2, "Reset Zoom", 'class="hide"');
 $im->stringTTF("search", $imagewidth - $xpad - 170, $fontsize * 2, "Search");
 $im->stringTTF("ignorecase", $imagewidth - $xpad - 75, $fontsize * 2, "Ignore-Case");
-$im->stringTTF("matched", $imagewidth - $xpad - 170, $imageheight - ($ypad2 / 2), " ");
+unless ($inverted) {
```
Comment (DanielShaulov) [inline flamegraph.pl:1262]: Use regular `if`, it is clearer when having an `else` clause

**Review by DanielShaulov — CHANGES_REQUESTED** [review]

**Comment by DanielShaulov** [comment]
Looks great in the image

File: flamegraph.pl (hunk 1 above)
Comment (rgruner) [reply flamegraph.pl:1262]: The entire codebase uses it like this (even with else clause)- just following conventions

Comment (DanielShaulov) [reply flamegraph.pl:1262]: OK

**Pull request merged**